contracts/gateway/GatewayBase.sol

```diff
@@ -521,6 +521,22 @@ contract GatewayBase is Organized {
         inboxActiveProcess[_account] = _messageHash;
     }
 
+    /**
+     * @notice Calculates the penalty amount for reverting a message transfer.
+     *
+     * @param _bounty The amount that facilitator has staked to initiate the
+     *                message transfers.
+     *
+     * @return penalty_ Amount of penalty needs to be paid by message initiator
+     *                  to revert message transfers.
+     */
+    function penalty(uint256 _bounty)
```

Comment:
I don't think `penalty` is a very good function name here. It lacks context. The function returns the penalty based on the bounty.
Maybe instead name it something like `penaltyFromBounty` or `bountyToPenalty` or `calculatePenalty` or something similar?